packages/cli/src/ui/commands/types.ts

```diff
@@ -196,6 +196,7 @@ export interface SlashCommand {
 
   // Optional metadata for extension commands
   extensionName?: string;
+  extension_id?: string;
```

Comment:
All of these other ones are camelCase, this should be too.

Also, isn't the ID just the name?